src/ansys/dpf/core/field.py

```diff
@@ -51,6 +51,14 @@ class Field(_FieldBase):
     The field's scoping defines the order of the data, for example: the first ID in the
     ``scoping`` identifies to which entity the first ``entity data`` belongs.
 
+    The minimum requirement for a well defined field is for it to have a dimensionality
+    (scalar, three components vector, six components symmetrical matrix, and so on), a location
+    ("Nodal", "Elemental", "ElementalNodal", "Timefrq"), a data vector and a scoping with IDs.
+    You can also set the number of shell layers. If the field has one elementary data by entity
+    (elementary data size equals the number of components for "Nodal" or "Elemental" field for example),
+    then the data vector can be set directly. If a more complex field is required
+    ("ElementalNodal" field for example), the data can be set entity by entity.
+
     For more information, see the `Fields container and fields
     <https://dpf.docs.pyansys.com/version/stable/user_guide/fields_container.html>`_
     documentation section.
```

Comment:
```suggestion
    For more information, see `Fields container and fields
    <https://dpf.docs.pyansys.com/version/stable/user_guide/fields_container.html>`_.
```
We just use the link (which will display the title in this case). No need to add descriptor text unless the link is to a page outside of the DPF-Core documentation.